sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1463,7 +1464,20 @@ def init(self):
         # Initialize xcvr table helper
         xcvr_table_helper = XcvrTableHelper()
 
-        self.load_media_settings()
+        fastboot_enabled = False
+        fastboot_tbl = swsscommon.Table(xcvr_table_helper.state_db_host, 'FAST_REBOOT')
```

Comment:
can we combine changes in line 1468 and 1472 in one function like here: [https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/device_info.py#L575](https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/device_info.py#L575) and use it to determine fast-reboot?